plugins/jobtype/src/azkaban/jobtype/HadoopSparkJob.java

```diff
@@ -283,8 +283,7 @@ private static void sparkJarsHelper(Props jobProps, String workingDir,
       Logger log, List<String> argList) {
     String jarList =
         HadoopJobUtils.resolveWildCardForJarSpec(workingDir, jobProps
-            .getString(SparkJobArg.SPARK_JARS.azPropName,
-                SparkJobArg.SPARK_JARS.defaultValue), log);
+            .getString(SparkJobArg.SPARK_JARS.azPropName, ""), log);
```

Comment:
Nit: better to have split arguments rather than the `.` operator:

```java
String jarList =
    HadoopJobUtils.resolveWildCardForJarSpec(workingDir,
        jobProps.getString(SparkJobArg.SPARK_JARS.azPropName, ""), log);
```